src/contracts/EthenaARM.sol

```diff
@@ -63,23 +71,44 @@ contract EthenaARM is Initializable, AbstractARM {
         _initARM(_operator, _name, _symbol, _fee, _feeCollector, _capManager);
     }
 
-    /**
-     * @notice Request a cooldown of USDe from Ethena's Staked USDe (sUSDe) contract.
-     * @param baseAmount The amount of base assets (sUSDe) to withdraw.
-     */
+    /// @notice Request a cooldown of USDe from Ethena's Staked USDe (sUSDe) contract.
+    /// @dev Uses a different unstaker contract each time to allow multiple cooldowns in parallel.
+    ///      There is a limit of MAX_REQUESTS_PER_DAY unstakers that can be used per day of the week.
+    /// @param baseAmount The amount of staked USDe (sUSDe) to withdraw.
     function requestBaseWithdrawal(uint256 baseAmount) external onlyOperatorOrOwner {
-        // Deploy a new EthenaUnstaker helper contract
-        EthenaUnstaker unstaker = new EthenaUnstaker(address(this), susde);
+        // Find which day of the week it is (0 = Thursday, 6 = Wednesday)
+        uint256 day = block.timestamp / 1 days % 7;
+        // Get last used unstaker for the day
+        uint8 index = lastUsedUnstakerIndex[uint8(day)];
+
+        // Cycle through unstakers for the day
+        // If never interacted with, start at the beginning of the day's unstakers
+        // If at the end of the day's unstakers, wrap around to the beginning
+        // Otherwise, just move to the next unstaker
+        if (index == 0 || index == (day + 1) * MAX_REQUESTS_PER_DAY - 1) {
+            index = uint8(day * MAX_REQUESTS_PER_DAY);
```

Comment:
Removing the day complexity should simplify this to something like
`index = lastUsedUnstakerIndex++ % MAX_REQUESTS`